packages/react-router/lib/router/router.ts

```diff
@@ -3676,7 +3679,7 @@ export function createStaticHandler(
         ? actionMatch
         : findNearestBoundary(matches, actionMatch.route.id);
 
-      let context = await loadRouteData(
+      let handlerContext = await loadRouteData(
```

Comment:
Rename for clarity:

* `handlerContext`-> a `StaticHandlerContext` returned from `staticHandler.query()`
* `requestContext` -> The `AppLoadContext` used passed to `staticHandler.query` by Remix SSR today
* `routerContext` -> This new `DefaultRouterContext` used for SPAs since it's not tired to a request like the SSR use case

In shared code, `requestContext` becomes `routerContext`